### PR TITLE
Display comments pop up

### LINKED ISSUE #14: [4pt]  -Display Comments pop up with selected item's details - Student B
Create a comments popup only with the top section (displaying details of the selected item)

Screenshot 2021-06-27 at 22 10 45

When the popup loads, the webapp retrieves data from the selected API and shows details about the selected item.

Add event to button prepared by your teammate that will open your popup. You need to communicate with your team member to make sure that you do not block each other (most likely the easiest solution is to merge the list of items feature to the dev branch before you try to add that event).

### PR DESCRIPTION
**Pull request by ClaudiaRojasSoto**
Dear @AndreaM2429 ,

I am pleased to submit my pull request to merge my 'display-comments-popup' branch into the 'develop' branch. This branch implements the following functionalities:

## Description:
- [X] Create a comments popup displaying details of the selected item.
- [X] The popup is created to show the necessary details of the selected item.
- [X] The data required for the popup is successfully retrieved from the corresponding API.
- [X] Add an event to the commits button (implemented by you) to open the popup.
- [X] Apply the necessary styling to the popup according to the design established on the homepage and the provided mockup by Microverse.

### HTML/CSS:
- [X] Used valid HTML syntax and appropriate semantic markup.
- [X] The HTML structure is well-organized and maintainable.
- [X] Meaningful and descriptive class and ID names have been employed.
- [X] Consistent and compatible styles have been applied throughout the project.

### JavaScript:
- [X] Followed the correct Gitflow for version control.
- [X] Documented my work in a professional manner.
- [X] The JavaScript code adheres to best practices, such as descriptive variable names and modularization when necessary.

I have attached my 'display-comments-popup' branch for your review. I appreciate any comments or suggestions you may have to improve both the functionality and the code. I am available to discuss any aspects related to this pull request and collaborate on any necessary adjustments.

Thank you for your time and attention. I look forward to your review and approval.

Best regards,
@ClaudiaRojasSoto

### CODE REVIEW

**Review by AndreaM2429 — CHANGES_REQUESTED** [review]
# **_God job so far_** 👏😄

@ClaudiaRojasSoto you made a great job! 🙌

But there is some suggestions that you need to implement before merge. 
Please check the comments below and don't hesitate to ask me any dubt or clarification about it.

File: src/Modules/commentsPopup.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  const popup = document.createElement('div');
+  popup.id = 'commentsPopup';
+  popup.innerHTML = `
+    <h2>${details.strMeal}</h2>
+    <img src="${details.strMealThumb}" alt="${details.strMeal}">
+    <p id="details-instructions">${details.strInstructions}</p>
+    <h3>Add a comment</h3>
+    <input type="text" placeholder="Your name" id="nameInput">
+    <textarea placeholder="Your comment" id="commentInput"></textarea>
+    <button class="closeButton">&times;</button>
```
Comment (AndreaM2429) [inline src/Modules/commentsPopup.js]: Please kindly add a `<form>` tag to contain the tags for the comments 👀
Also please try to make only the requerements, this part is for the next task.

Comment (ClaudiaRojasSoto) [reply src/Modules/commentsPopup.js]: Thank You Andrea for your comments and very useful suggestion. You are right. The form wasn't a requirement for this task. What do you think, do you think I should delete that part then?

Comment (AndreaM2429) [reply src/Modules/commentsPopup.js]: Is not necessary to delete it, please just add the `<form>` and be careful for the next time. 😊

Comment (ClaudiaRojasSoto) [reply src/Modules/commentsPopup.js]: Ok, I will do it right now. Thank you for your time!!

**Review by AndreaM2429 — APPROVED** [review]
# **Status: Approved 🟢**

Your project is completed, there is nothing else to say so is time to merge it.🐱‍👤

congrats

**Pull request merged**